configuration/configure-storage-management.md

```diff
@@ -0,0 +1,33 @@
+---
+description:  This article describes how you can set up Storage Management in Syskit Point. 
+---
+
+# Storage Management
+
+The Syskit Point Storage Management feature helps you keep your environment clutter-free, optimize the usage of your storage, and gain insight into how your storage is distributed across your tenant.
+
+For more information on this feature, [take a look at the Optimize Storage article](../governance-and-automation/optimize-storage.md). 
+
+To set up the Storage Management feature within Syskit Point, complete the following steps:
+
+  * **Go to the Settings (1)** of your Syskit Point
+  * Under the General section, **click the Storage Management button (2)**
+  * **Click the Collect storage data checkbox (3)** to turn on the storage management feature
+
+Once storage management has been turned on, you can also customize your storage reports in Syskit Point by setting the following:
+  * **Show how much I can save if I keep the latest X file versions (4)** - here, you can set a number in place of X depending on how many of the latest file versions you want to keep when trying to declutter your workspaces
```

Comment:
kod ovih opcija treba bolje objasniti gdje će se to odraziti - na kojem točno reportu/reportima

Reply:
za postavljanje ove dvije vrijednosti u settingsima .. to se vidi na Site Storage Metrics reportu - na tileu File Version Cleanup i na File Storage Details reportu na dva tilea s kojim se može filtirati verzije